lib/index.js

```diff
@@ -240,11 +240,23 @@ const init = (configOverrides = {}) => {
   // f: notify system that a file has changed
   _.fileChanged = (absolutePath) => {
     _.log(`file-changed: ${path.relative(config.directories.root, absolutePath)}`);
+
```

Comment:
Hmmmm. Why are you busting the cache here?

Check out https://github.com/alexisvincent/systemjs-tools/blob/master/lib/index.js#L346
and https://github.com/alexisvincent/systemjs-tools/blob/master/lib/index.js#L96.